agent/proto/agentpb/agent.proto

```diff
@@ -167,28 +167,32 @@ message TimelineEvent {
   
   // Data contains event specific data.
   oneof data {
+    // UnknownEvent indicates a unknown event.
+    UnknownEvent unknownEvent = 2;
```

Comment:
Usually, the protobuf attributes are not re-arranged after creation - you can remove bits but leave the holes (aka `reserved` fields) and add. Do you think this might affect the timeline evaluation when the cluster is upgraded (i.e. when the old/new ordering exist at the same time)?

Reply:
Yeah, if satellite is not updated in planet and gravity at the same time the timeline evaluation will be incorrect. I guess it too late to change the ordering here. Will revert.